### PR TITLE
Created NavItem component

### PR DESCRIPTION
**Pull request by gunbux**
### Changes made
- Added NavItem

### CODE REVIEW

File: src/components/styles/NavItem.styled.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        background-position: 50% 100%;
+        // change this to adjust width and thickness out underline
+        background-size: 60% 2px;
+      `
+    }
+    return `
+      cursor: pointer;
+      &:hover {
+        font-size: 25px;
+  }
```
Comment (rldefa) [inline src/components/styles/NavItem.styled.ts:28]: indentation triggers ocd :0

File: src/components/styles/NavItem.styled.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  justify-content: center;
+  align-items: center;
+  font-size: 22px;
+  transition: font-size 0.3s;
+  text-transform: capitalize;
+  ${(props) => {
+    if (props.isActive) {
+      return `
+        color: ${props.theme.palette.primary};
+        //css solution for styled underline
```
Comment (rldefa) [inline src/components/styles/NavItem.styled.ts:16]: 👍  sick

File: src/components/NavItem.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
+/* eslint-disable react/prefer-stateless-function */
```
Comment (rldefa) [inline src/components/NavItem.tsx:1]: slightly sus 🤔

**Review by rldefa — APPROVED** [review]
lgtm

File: src/components/NavItem.tsx (hunk 3 above)
Comment (gunbux) [reply src/components/NavItem.tsx:1]: mb on this one, was testing something and forgot to remove the disable

**Review by zzhikai — APPROVED** [review]
lgtm

**Comment by gunbux** [comment]
[Nav item](https://trello.com/c/q8nJhewe/5-nav-item)

**Pull request merged**